lms/templates/wiki/base.html

```diff
@@ -1,4 +1,5 @@
 {% extends "main_django.html" %}
+{% with online_help_token="wiki" %}
```

Comment:
Why does this file use a `with`/`endwith` construct instead of the `<%def name="online_help_token()"><% return "wiki" %></%def>` construct used in the other files?